src/style.css

```diff
@@ -1,3 +1,69 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  align-items: center;
+  line-height: 60px;
+  background: white;
+}
+
+.nav__list {
+  margin: 0;
+  display: flex;
+  list-style-type: none;
+  padding: 0;
+}
```

Comment:
Don't use tag names for styling (except `html` and `body`). Style all elements using `.class` and if needed with `:pseudo-class`, `pseudo-element` and `[attribute]`. Instead of `.nav__list`, you should define styles for `ul` elements within the `.nav` class like `.nav ul`.